README.md

```diff
@@ -364,7 +364,7 @@ const fly = new Fly({
 
 ### Task
 
-A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of convention; of course, you may call it whatever you'd like.
+A Task receives itself as its first argument. We choose to name the parameter `fly` simply because of the convention; of course, you may call it whatever you'd like.
```

Comment:
Well, yeah. The `the` **seems** to be referring to some ubiquitous and insurmountable convention followed by everyone.

"simply as a convention" sounds about right.